tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,27 @@
+import os
+import filecmp
+
+from openmc import _utils
+import pytest
+
+@pytest.fixture()
+def download_photos(run_in_tmpdir):
+    """use _utils download() function to download the same picture three times,
+       twice to get unique names, & a third time to use the already downloaded
+       block of code"""
+    _utils.download("https://i.ibb.co/HhKFc8x/small.jpg")
+    _utils.download("https://tinyurl.com/y4t38ugb")
+    _utils.download("https://tinyurl.com/y4t38ugb", as_browser=True)
+
+
+@pytest.fixture()
+def get_checksum_error(run_in_tmpdir):
```

Comment:
This is actually not getting run because it is set up as a fixture, and the fixture is not actually used by any tests. I'd suggest just removing the `@pytest.fixture` decorator and changing the function name to `test_checksum_error`.

Reply:
Fixed, thanks for being attentive!